app/scripts/controllers/RoomCtrl.js

```diff
@@ -0,0 +1,13 @@
+(function() {
+    function RoomCtrl(Room) {
+        this.title = "Here's RoomCtrl!";
+        
+        // something magical goes here
+        this.rooms = Room;
```

Comment:
Perhaps make this `this.allRooms = Room.all;`, then in the template you can ng-repeat on `rooms.allRooms`.